gocd/pipelines/relay.yaml

```diff
@@ -82,7 +82,7 @@ pipelines:
           # Manually completes this pipeline run.
           # This will trigger relay-pop deployments.
           approval:
-            type: manual
+            type: success
```

Comment:
Does this mean PoPs will automatically deploy if the Relay deploy was successful?

Reply:
Nope, since deploy for PoPs will have to be approved manually again, see 
https://github.com/getsentry/relay/blob/a4290b4d53bce5781d80b1b3ff1be88b234b8240/gocd/pipelines/relay-pop.yaml#L27-L29